src/components/Form/Form.tsx

```diff
@@ -141,17 +151,22 @@ const Form = <T extends FormDetails>({
           </div>
           <div className="flex flex-col-reverse justify-end gap-3 sm:flex-row">
             {!hideCancelButton && (
-              <Cancel
+              <ButtonV2
+                variant="secondary"
                 onClick={handleCancel}
-                label={props.cancelLabel ?? "Cancel"}
-              />
+                disabled={disabled}
+              >
+                {props.cancelLabel ?? t("common:cancel")}
+              </ButtonV2>
             )}
-            <Submit
-              data-testid="submit-button"
+            <ButtonV2
+              variant="primary"
```

Comment:
Make sure the data-testid is not being used in tests first, before removing them.